### ISSUE TITLE
R503 false positive

### ISSUE THREAD

**Issue by jsh9** [Afonasev/flake8-return#134]
* Date you used flake8-return: 5/18/2023
* flake8-return version used, if any: 1.2.0
* Python version, if any: 3.8
* Operating System: macOS

### What I Did

For this type of function:
```python
def func(arg1) -> int:
    if arg1 == 'something':
        return 2

    if arg1 == 'something else':
        return 3

    raise ValueError('Invalid arg1 value')
```

flake8-returns gives R503 violation.  I think this may be a false positive
